inst/sympref.m

```diff
@@ -369,14 +369,14 @@
 %% Test for correct line numbers in Python exceptions.  These first tests
 %% happen with the default ipc---usually popen2.  We don't explicitly test
 %% popen2 because test suite should be portable.
-%!error <line 1> python_cmd('raise ValueError');
-%!error <line 1> python_cmd('raise ValueError', sym('x'));
-%!error <line 1> python_cmd('raise ValueError', sym([1 2 3; 4 5 6]));
-%!error <line 1> python_cmd('raise ValueError', {1; 1; 1});
-%!error <line 1> python_cmd('raise ValueError', struct('a', 1, 'b', 'word'));
-%!error <line 2> python_cmd( {'x = 1' 'raise ValueError'} );
-%!error <line 3> python_cmd( {'x = 1' 'pass' '1/0'} );
-%!error <line 3> python_cmd( {'a=1' 'b=1' 'raise ValueError' 'c=1' 'd=1'} );
+%!error <ValueError> python_cmd('raise ValueError');
```

Comment:
these tests are specifically designed to check that the line numbers are correct.  I won't merge this.

I suppose we could drop the line number feature...